scripts/generate_virtual_player.py

```diff
@@ -0,0 +1,40 @@
+import numpy as np
+import random
+import os
+
+from player import VirtualPlayer
+from game_session import GameSession
+from models import RescorlaWagner, Qlearning
+
+def prepare_parameters(number_of_trials, num_of_params):
+    parameters = np.zeros((number_of_trials, num_of_params))
+    if num_of_params == 3:
+        for params in range(parameters.shape[0]):
```

Comment:
notice than you have a separate variable for expression parameters.shape[0],
if statement is redundant as you use the same distribution both for case when 2 parameters are needed and for the case with 3, you could just use slicing e.g parameters[params][1:] = random.uniform(). Notice also that naming index of array "params" is somewhat misleading for the reader.